### PR TITLE
radioGroup 新增配置调用方式 并新增 demo

### PR DESCRIPTION
**Pull request by ascoders**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you propose PR to correct branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: components/radio/demo/radiogroup-optional.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  en-US: RadioGroup group - optional
+---
+
+## zh-CN
+
+一组互斥的 Radio 配合使用。
+
+## en-US
+
+A set of mutually exclusive Radio with the use of
```
Comment (afc163) [inline components/radio/demo/radiogroup-optional.md]: Don't use Google translate.

Comment (ascoders) [inline components/radio/demo/radiogroup-optional.md]: I just copied another demo and forgot to change it.

Comment (afc163) [inline components/radio/demo/radiogroup-optional.md]: Then it is our translation mistake...

**Comment by afc163** [comment]
You should PR to branch `feature-2.9` cause it is a new feature.

**Comment by ascoders** [comment]
@afc163 How can I submit this PR to branch `feature-2.9`?  Do I have to submit a new pr?

**Comment by ascoders** [comment]
to #5461

**Pull request closed without merging**